src/css/style.css

```diff
@@ -1,4 +1,91 @@
-/* Your custom styles go here! */
+:root {
+    --grey-lightest: #f7f9fa;
+    --blue-light: rgba(43, 63, 86, 0.81);
+    --green: #72cc18;
+    --green-light: rgba(114, 204, 24, 0.68);
+}
+
+html, body {
+    margin: 0;
+    height: 100%;
+    font-family: "Helvetica Neue", Sans-Serif, sans-serif;
+    background: var(--blue-light);
+    color: var(--grey-lightest);
+}
+
+body a {
+    color: orange;
+}
+
+body a:hover {
+    color: darkorange;
+}
+
+ul {
+    list-style-type: square;
+    list-style-position: inside;
+}
+
+#participants li {
+    border-bottom: solid 1px darkorange;
+    margin-bottom: 2px;
+    padding: 1px;
+    color: var(--grey-lightest);
+    border-radius: 3px;
+}
+
 #container {
     margin-top: 52px;
+    height: 100%;
+}
+
+#call {
+    display: flex;
+    flex-direction: row;
+    height: 100%;
+}
+
+#callFrame {
+    width: 70%;
+    height: 100%;
```

Comment:
This should be lower than 100% because it doesn't factor in the header height, so it's pushing the controls off the screen. 
`height: calc(100% - 52px);` seems to fix it (you can set the header height to a variable and use that in the header styles and here to avoid the "magical" number)

Reply:
(controls = prebuilt tray)